code/game/objects/structures/fullwindow.dm

```diff
@@ -78,15 +78,15 @@
 	desc = "It looks rather strong and frosted over. Looks like it might take a few less hits then a normal reinforced window."
 	icon_state = "fwindow"
 	basestate = "fwindow"
-	health = 30
+	health = 60
 
 /obj/structure/window/full/shuttle
 	name = "shuttle window"
 	desc = "It looks rather strong. Might take a few good hits to shatter it."
 	icon = 'icons/obj/podwindows.dmi'
 	icon_state = "window"
 	basestate = "window"
-	health = 160
+	health = 320
```

Comment:
Why are you doubling shuttle window health? Even when the other windows get double health shuttle windows are the strongest. Also there is no non fulltile shuttle window

Reply:
Consistency mostly, its not simply "oh, its got a weaker version" but rather a "still the same strength as the plasma reinforced window it previously matched"

Reply:
I guess you could leave it for consistency, but this will make shuttle windows take twice as long to break (they already take quite long). With a common item like a fire extinguisher it'll take 32 hits as opposed to the previous, if the pods or shuttles are about to leave being able to break in from space and quickly board them will become impossible. Honestly don't think non plasma windows should be that strong

Reply:
You can still un-anchor shuttle windows. If you are trying to break in via brute force, that's on you to make the choice.

Reply:
I know you can unanchor them with a screwdriver+crowbar only, this is still a huge buff to shuttle windows, taking 16 hits with a spear instead of 8 and 32 hits with a fire extinguisher instead of 16. You can't even build shuttle windows unlike the other windows you doubled the health of. Shuttle windows are not buildable, don't have a directional equivalent, so why double their health? This seems like a buff for the sake of buffing it, there's no hard rule that shuttle windows have to be equivalent to plasma reinforced windows. And shuttle windows aren't just used in shuttles, they're used in escape pods too. Plasma windows are meant for use with dangerous atmos like Toxins Mixing

Reply:
Escape pods are shuttles, though at this point this is more arguing semantics than anything.

They got the health buff for consistency, plain and simple. Your disagreement with this change has been noted, but at this time I see no reason to undo that portion of the PR.

Yes, the shuttle window does not have a directional counterpart (honestly doesn't need them in my opinion), and cannot be built. The same, however, applies to shuttle walls and floors which are actually unable to be deconstructed or constructed. Players aren't meant to be building shuttle stuff (at least at this time), so the fact that you can't build a shuttle window honestly doesn't feel like much of a valid concern in regards to the health of the window.

Reply:
I agree with uraniummeltdown- shuttle windows shouldn't be ridiculously tough, the fact that they previously matched plasma glass shows that plasma glass had the incorrect values, that's about it.

Reply:
In that case, what SHOULD they be at? I don't want to make a bunch of blind changes on this number until I find one you are fine with.

Reply:
Leave them at 160, they're fine where they are now.

Reply:
Alright, I'll make that change tonight, I guess that will leave them between a reinforced glass and normal plasmaglass window anyways.